src/kata7.getEmployerRole.js

```diff
@@ -1,3 +1,5 @@
-const getEmployerRole = (employeeName, employees) => {};
-
+const getEmployerRole = (employeeName, employees) => {
+    let newObject =  employees.filter((element) => element.name === employeeName);
```

Comment:
Is the data type of newObject here an object or an array? Also does it need to be let, or could it be const?